### ISSUE TITLE
Document how to train with large batch sizes

### ISSUE THREAD

**Issue by tfboyd** [tensorflow/tensorflow#22358]
This request was a result of the TensorFlow Fall Symposium. Document tips and methods to use for large batch scaling with TensorFlow.  Possibly mention where this is proven and unproven to work.  This would include adding optimizers or wrapper to simplify usage.  Researchers are large labs want to scale but do not have the knowledge to do it quickly.

**Comment by tfboyd** [comment]
Randomly assigned, feel free to reassign.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @guptapriya: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @guptapriya: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @guptapriya: It has been 44 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @guptapriya: It has been 62 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
